From a411a26ed52d23bdef25a7c1996513ef09e64c29 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Wed, 22 Feb 2012 13:23:03 +0530
Subject: [PATCH 092/182] ARM: OMAP2+: nand: Flag to support ELM feature on
 platforms.

This patch introduces a platform data flag to selectively enable/disable
ELM feature for NAND. On platforms where ELM support is present, this
flag needs to be set as true.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/plat-omap/include/plat/nand.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-omap/include/plat/nand.h b/arch/arm/plat-omap/include/plat/nand.h
index 67fc506..1bcff1d 100644
--- a/arch/arm/plat-omap/include/plat/nand.h
+++ b/arch/arm/plat-omap/include/plat/nand.h
@@ -29,6 +29,7 @@ struct omap_nand_platform_data {
 	unsigned long		phys_base;
 	int			devsize;
 	enum omap_ecc           ecc_opt;
+	bool			elm_used;
 };
 
 /* minimum size for IO mapping */
-- 
1.7.9.1

